pkg/controller/stack/session_test.go

```diff
@@ -3,6 +3,7 @@ package stack
 import (
 	"errors"
 	"fmt"
+	"github.com/pulumi/pulumi-kubernetes-operator/pkg/logging"
```

Comment:
Given the existing ordering, this import should be on L12 if I'm not mistaken

Reply:
We typically adhere to `go fmt` which is admittedly a little weird about it. Still prefer it to `go imports` which does some arbitrary newline sprinkling.